gocd/bci_repo_publish.py

```diff
@@ -166,9 +186,13 @@ def run(self, version, token=None):
             }
             url = makeurl(self.apiurl, ['trigger', 'release'], params)
             # No bindings for using tokens yet, so do the request manually
-            req = requests.post(url, headers={'Authorization': f'Token {token}'})
-            if req.status_code != 200:
-                raise RuntimeError(f'Releasing failed: {req.text}')
+            req = requests.post(url, headers={'Authorization': f'Token {bci_repo_token}'})
+            req.raise_for_status()
+
+        for pkg in self._fetch_package_names_in_project(build_prj):
+            url = makeurl(self.apiurl, ['trigger', 'rebuild'], {'project': build_prj, 'package': pkg})
```

Comment:
I assume the project wide retrigger didn't work?

Reply:
No: https://github.com/openSUSE/open-build-service/issues/15114